generators/server/templates/build.gradle.ejs

```diff
@@ -262,7 +262,15 @@ repositories {
     jcenter()
     <%_ if (reactive && databaseType === 'sql' || JHIPSTER_DEPENDENCIES_VERSION.endsWith('-SNAPSHOT')) { _%>
     maven {
+        <%_ if (process.env.JHI_GITHUB_CI === 'true') { _%>
+        credentials {
```

Comment:
one another reason I'm not convinced about this approach is to generate specific code for our CI. 

It means it's different from local / CI

Reply:
Maven could be configured with a custom settings.xml.
Don’t know about Gradle.